src/ansys/motorcad/core/geometry.py

```diff
@@ -516,6 +545,57 @@ def get_entities_have_common_coordinate(entity_1, entity_2):
         return False
 
 
+def entities_same(entities_a, entities_b):
```

Comment:
Need to add these to docs manually in: \doc\source\methods\geometry_functions.rst